File: 2298/2298.c
int tasksCountInTheWeekend(struct TreeNode* root) {
    if (!root) return 0;
    int l = tasksCountInTheWeekend(root->left);
    int r = tasksCountInTheWeekend(root->right);
    return 1 + (l > r ? l : r);
}

File: 2298/2298.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var tasksCountInTheWeekend = function(root) {
    if (!root) return 0;
    return 1 + Math.max(tasksCountInTheWeekend(root.left), tasksCountInTheWeekend(root.right));
};